docs/GettingStarted.md

```diff
@@ -33,10 +29,9 @@ Your feedback on the beta would be greatly appreciated. If you have any issues,
 Profiles are JSON files that describe the data you want to generate. They are composed of:
 
 -   **fields** - an array of uniquely named fields (or columns).
--   **rules** - an array of constraints, with an optional description.
--   **constraints** - restrict the types and ranges of data permitted for the given column.
+-   **constraints** - an array of restrictions on the types and ranges of data permitted for the given column.
 
-We'll start by generating data for a trivial schema. Using your favourite text editor, create the following JSON profile and save it as `profile.json`:
+We'll start by generating data for a trivial schema. Using [VS Code](https://code.visualstudio.com/) or your favourite text editor, create the following JSON profile and save it as `profile.json`:
```

Comment:
Do we really need to recommend a text editor to people?

Reply:
I put this in because in a later section, (profile validation), we go through how to setup profile validation on VS code specifically.

Would you like me to remove this?

Reply:
I have removed this